diff -dur -x '*~' muse-3.0.2.orig/muse/widgets/genset.h muse-3.0.2/muse/widgets/genset.h
--- muse-3.0.2.orig/muse/widgets/genset.h	2017-12-04 22:01:19.000000000 +0100
+++ muse-3.0.2/muse/widgets/genset.h	2018-11-22 19:13:29.182616803 +0100
@@ -29,6 +29,7 @@
 #include "cobject.h"
 #include "mdisettings.h"
 
+#include <QButtonGroup>
 #include <QShowEvent>
 #include <list>
 
diff -dur -x '*~' muse-3.0.2.orig/muse/widgets/pastedialog.h muse-3.0.2/muse/widgets/pastedialog.h
--- muse-3.0.2.orig/muse/widgets/pastedialog.h	2017-12-04 22:01:19.000000000 +0100
+++ muse-3.0.2/muse/widgets/pastedialog.h	2018-11-22 19:13:48.033847674 +0100
@@ -25,6 +25,7 @@
 
 #include "ui_pastedialogbase.h"
 #include <QString>
+#include <QButtonGroup>
 
 namespace MusECore {
 class Xml;
diff -dur -x '*~' muse-3.0.2.orig/synti/simpledrums2/simpledrumsgui.h muse-3.0.2/synti/simpledrums2/simpledrumsgui.h
--- muse-3.0.2.orig/synti/simpledrums2/simpledrumsgui.h	2018-01-29 21:07:03.000000000 +0100
+++ muse-3.0.2/synti/simpledrums2/simpledrumsgui.h	2018-11-22 19:10:58.517743603 +0100
@@ -35,6 +35,7 @@
 #include <QGroupBox>
 #include <QHeaderView>
 #include <QComboBox>
+#include <QButtonGroup>
 #include <widgets/meter.h>
 
 #include "libsynti/gui.h"
--- muse-3.0.2.orig/muse/mplugins/mitplugin.cpp	2017-12-04 22:01:18.000000000 +0100
+++ muse-3.0.2/muse/mplugins/mitplugin.cpp	2018-11-22 19:48:54.657286284 +0100
@@ -35,6 +35,8 @@
 #include "rhythm.h"
 #endif
 
+#include <QAction>
+
 namespace MusEGui {
 
 //---------------------------------------------------------
