--- muse/widgets/CMakeLists.txt.old	2013-03-28 16:17:35.000000000 +0100
+++ muse/widgets/CMakeLists.txt	2013-06-01 19:34:33.000000000 +0200
@@ -271,6 +271,7 @@
 ##
 target_link_libraries ( widgets
       ${QT_LIBRARIES}
+      X11
       icons
       instruments
       )
--- CMakeLists.txt.org	2013-06-01 19:43:43.000000000 +0200
+++ CMakeLists.txt	2013-06-01 19:45:23.000000000 +0200
@@ -86,7 +86,7 @@
 SET(MusE_VERSION_PATCH 0)
 SET(MusE_VERSION       "2.1.2")
 SET(MusE_VERSION_FULL  "2.1.2")
-SET(MusE_INSTALL_NAME  "muse-2.1")
+SET(MusE_INSTALL_NAME  "muse")
 SET(MusE_EXEC_NAME     "muse2")
 
 ## The directory where we will install the shared components:
